pywr/notebook/draw_graph.js

```diff
@@ -136,6 +137,12 @@ require(["d3"], function(d3) {
 
     function tick() {
 
+        node.attr("transform", function(d) {
```

Comment:
Add a comment that this is bounding the node positions to stay inside the box. 

An alternative to this is to retain the coordinates as is, but use a `viewBox` on the SVG element to view the box containing the model?